app/helpers/application_helper/toolbar/configuration_profile_center.rb

```diff
@@ -0,0 +1,20 @@
+class ApplicationHelper::Toolbar::ConfigurationProfileCenter < ApplicationHelper::Toolbar::Basic
+  button_group('access', [
+    select(
+      :access_choice,
+      nil,
+      N_('Access'),
+      N_('Access'),
+      :items => [
+        button(
+          :configuration_profile_console,
```

Comment:
@k-chu you will also need to add RBAC feature for this new toolbar button in core repo under https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_product_features.yml#L4892 section

Reply:
Added RBAC - https://github.com/ManageIQ/manageiq/pull/20646